src/main/java/net/snowflake/ingest/streaming/internal/BlobMetadata.java

```diff
@@ -18,24 +18,33 @@ class BlobMetadata {
   private final Constants.BdecVersion bdecVersion;
   private final List<ChunkMetadata> chunks;
   private final BlobStats blobStats;
+  private final boolean spansMixedTables;
```

Comment:
want to add a comment explain this? The name is not straight forward like other fields

Reply:
Or I suggest we call it something like `hasMultipleTables`

Reply:
I added a comment.